src/storage/Cargo.toml

```diff
@@ -37,7 +37,7 @@ mz-expr = { path = "../expr" }
 mz-interchange = { path = "../interchange" }
 mz-kafka-util = { path = "../kafka-util" }
 mz-kinesis-util = { path = "../kinesis-util" }
-mz-ore = { path = "../ore", features = ["task", "tracing_"] }
+mz-ore = { path = "../ore", features = ["ssh", "task", "tracing_"] }
```

Comment:
I hit this too, if you cargo check `src/storage` in isolation it fails because this feature is missing. When you compile from the root of the repo it works by accident because some other package enables the feature